packages/mdc-toolbar/README.md

```diff
@@ -72,47 +125,144 @@ toolbar will not overlay any of the element's content.
 </main>
 ```
 
-### Sections
 
-Toolbar sections are aligned to the toolbar's center. You can change this
-behavior by applying `mdc-toolbar__section--align-start` or
-`mdc-toolbar__section--align-end` to align the sections to the start or the end
-of the toolbar (respectively).
+### Waterfall toolbars (Requires JavaScript)
+
+When you are using `mdc-toolbar--fixed` with JavaScript. Toolbars will be upgraded
```

Comment:
I think you are right that `fixed` and `waterfall` seems 2 different behaviors, which I think I misunderstand in the first place. However, I think the above solution doesn't take `fixed-last-row-only` into consideration. So I would suggest that move both `waterfall` as well as `fixed-last-row-only` as modifier for `fixed` instead, which became:
- fixed: `class="mdc-toolbar--fixed"`
- fixed + waterfall: `class="mdc-toolbar--fixed mdc-toolbar--waterfall"`
- fixed + fixed-last-row-only: `class="mdc-toolbar--fixed mdc-toolbar--fixed-last-row-only"`
- fixed + waterfall + fixed-last-row-only: `class="mdc-toolbar--fixed mdc-toolbar--fixed-last-row-only mdc-toolbar--waterfall"`

Does this sounds good to you?